packages/editor/src/plugins/edusharing-asset/renderer.tsx

```diff
@@ -32,6 +31,12 @@ const EmbedJson = t.type({
   }),
 })
 
+const iframeResizerHtml =
+  '<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.contentWindow.min.js"></script>'
+
+// Set some styles to default values to create consistency between different browsers similar to a CSS reset, see https://meyerweb.com/eric/tools/css/reset/
+const defaultStyle = 'padding: 0; margin: 0; border: 0;'
```

Comment:
Imo, CSS reset is widely known enough for this comment to be unnecessary. Maybe consider:
```suggestion
const cssReset = 'padding: 0; margin: 0; border: 0;'
```

Reply:
I changed it. :slightly_smiling_face: